packages/turbo/ios/RNVisitableView.swift

```diff
@@ -126,16 +126,31 @@ class RNVisitableView: UIView, RNSessionSubscriber {
       onVisitProposal!(event)
     }
   }
+    
+  func getStatusCodeFromError(error: TurboError?) -> Int {
```

Comment:
I'm unable to extend `TurboError` enum, so I couldn't match Android style (I also wanted to avoid maintaining two identical enums on iOS, it would be better to inherit from it but it's impossible)